src/WebJobs.Script.WebHost/Management/GetTriggersResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using static Microsoft.Azure.WebJobs.Script.WebHost.Management.FunctionsSyncManager;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public class GetTriggersResult
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the sync operation was successful.
+        /// </summary>
+        public bool Success { get; set; }
+
+        /// <summary>
+        /// Gets or sets the error string in the case of a failed sync operation.
+        /// </summary>
+        public string Error { get; set; }
+
+        /// <summary>
+        /// Gets or sets the payload of the triggers.
+        /// </summary>
+        public SyncTriggersPayload Payload { get; set; }
```

Comment:
I think rather than using this type here, you should just have a "string Content" member. I'm not sure why we ever added the SyncTriggersPayload.Count member - looks like it's only used internally by FunctionsSyncManager. So Count doesn't need to be part of our external contract I think. Note that we only pass Content to FE on syncs anyways.

Reply:
I saw this block of comments for sync trigger method:
```
// We don't do background sync for empty triggers.
// We've seen error cases where a site temporarily gets into a situation
// where it's site content is empty. Doing the empty sync can cause the app
// to go idle when it shouldn't.
```

@mathewc - The question is: If we get 0 count, should we overwrite the empty payload during deployment or not? 
If we should overwrite, then I can remove count.
If we should not, then we will need the count.

Reply:
You don't need the count regardless, since it's just the length of the triggers array which is already part of the Content. I actually don't remember why we added the Count member.

Reply:
Oh yes, the content is not encrypted. Updated GetTriggersResult to:

```
public class GetTriggersResult : TriggersOperationResult
{
    /// <summary>
    /// Gets or sets the triggers payload.
    /// </summary>
    public string Content { get; set; }
}